tests/features/search_and_filters/search_and_filters.spec.ts

```diff
@@ -0,0 +1,163 @@
+import { expect } from '@playwright/test';
+import { test } from '../../config/test-base';
+
+const datasourceFilter = 'Datasource';
+const namespaceFilter = 'Namespace';
+const ownerFilter = 'Owner';
+const typeFilter = 'Type';
+const tagFilter = 'Tag';
+const groupsFilter = 'Groups';
+const datasourceOptionAirFlights = 'Air_flights';
+const datasourceOptionBookETL = 'Bookshop ETL';
+const namespaceOption = 'ETL';
+const entityNameWithSpecialChars = '_!+AirFlights';
+const bookETLDataEntity = 'BookETL';
+const bookETLDataAQAEntity = 'Book_ETL_aqa';
+const kdsClickstreamDataEntity = 'kds_clickstream_consumer';
+const postcardsAQADataEntity = 'postcards_aqa';
+const tagFilterOption = 'aqa_tag';
+const groupsOption = 'group_bookshop';
+const transformersTag = 'Transformers';
+const firstOwner = 'firstOwner';
+const titleAdmin = 'admin';
+const noResultsOwnerText = 'No result. Create new owner "firstOwner"';
+const noResultsTitleText = 'No result. Create new title "admin"';
+const noResultsTagText = 'No result. Create new tag "aqa_tag"';
+
+test.describe('Check filters', () => {
+  test.describe('when apply single filter', () => {
+    test.beforeEach(async ({ steps: { pages } }) => {
+      await test.step(`I open catalog page`, async () => {
+        await pages.topPanel.clickTab('Catalog');
+      });
+    });
+    /**
+     * /project/1/test-cases/23
+     */
+    test('should display the expected item with Datasource filter', async ({
+      steps: { pages },
+    }) => {
+      await test.step('apply filter Datasource', async () => {
+        await pages.catalog.openFilterWithSelect(datasourceFilter);
+        await pages.catalog.chooseOption(datasourceOptionAirFlights);
+        expect(await pages.catalog.isListItemVisible(entityNameWithSpecialChars)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/24
+     */
+    test('should display the expected item with Namespace filter', async ({ steps: { pages } }) => {
+      await test.step('apply filter Namespace', async () => {
+        await pages.catalog.openFilterWithSelect(namespaceFilter);
+        await pages.catalog.chooseOption(namespaceOption);
+        expect(await pages.catalog.isListItemVisible(entityNameWithSpecialChars)).toBeTruthy();
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+        expect(await pages.catalog.isListItemVisible(bookETLDataAQAEntity)).toBeTruthy();
+        expect(await pages.catalog.isListItemVisible(kdsClickstreamDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/25
+     */
+    test('should display the expected item with Owner filter', async ({ steps: { pages } }) => {
+      await test.step('apply filter Owner', async () => {
+        await pages.catalog.searchByText(bookETLDataEntity);
+        await pages.catalog.clickOnListItem(bookETLDataEntity);
+        await pages.overview.createOwner(
```

Comment:
why do we create a method with 4 arguments? As I understand, 2 arguments should be enough for method which creates new owner

Reply:
These 4 arguments are used for creating owner and title, because firstly we add some name and title to inputs and if there are no some matches - we click to select offer to create the owner and the title